modules/ppcp-wc-gateway/src/Processor/OrderProcessor.php

```diff
@@ -160,12 +162,22 @@ public function __construct(
 	 *
 	 * @return bool
 	 */
-	public function process( \WC_Order $wc_order ): bool {
-		$order_id = $wc_order->get_meta( PayPalGateway::ORDER_ID_META_KEY );
-		$order    = $this->session_handler->order() ?? $this->order_endpoint->order( $order_id );
+	public function process( WC_Order $wc_order ): bool {
+		$order = $this->session_handler->order();
 		if ( ! $order ) {
-			$this->last_error = __( 'No PayPal order found in the current WooCommerce session.', 'woocommerce-paypal-payments' );
-			return false;
+			$order_id = $wc_order->get_meta( PayPalGateway::ORDER_ID_META_KEY );
+			if ( ! $order_id ) {
+				$this->logger->warning( "No PayPal order ID found in order #{$wc_order->get_id()} meta." );
```

Comment:
I think it must be `$`, not `#`?

Reply:
yeah, that's a typo, the idea was to render it as "in order #123 meta"